tools/releases/helm.sh

```diff
@@ -16,13 +16,22 @@ GH_PAGES_BRANCH="gh-pages"
 function package {
   # First package all the charts
   for dir in "${CHARTS_DIR}"/*; do
-    if [ ! -d "$dir" ]; then
+    if [ ! -d "${dir}" ]; then
       continue
     fi
 
+    git diff HEAD --exit-code -- "${dir}"
```

Comment:
```suggestion
    git diff --exit-code HEAD -- "${dir}"
```

Also worth a comment ... I think that you are just trying to ensure there are no uncommitted changes here?

Reply:
Yep